lib/Pipes/AllFunctions.cpp

```diff
@@ -0,0 +1,48 @@
+//
```

Comment:
Missing proper header for `.cpp` files.

Reply:
Done!

Just so you know, in `revng-c` we're not using either of those on top of `.cpp` files. I think we never did, or if we did we got rid of them a while back. You might want to take that into consideration when merging the two repositories in the future.